migrations/settings/0001-migrate-staff-setting.js

```diff
@@ -0,0 +1,9 @@
+export default function migrate(settings) {
+  if (
+    settings.has("show_only_for_staff") &&
+    settings.get("show_only_for_staff")
+  ) {
+    settings.set("limit_to_groups", "staff");
```

Comment:
I'm slightly confused here, don't we store the group ids in the theme setting instead of the group's name?